Signal/src/call/CallService.swift

```diff
@@ -1511,44 +1548,16 @@ protocol CallServiceObserver: class {
 
         Logger.debug("\(self.logTag) in \(#function)")
 
-        self.localVideoTrack = nil
-        self.remoteVideoTrack = nil
-        self.isRemoteVideoEnabled = false
+        // Capture a reference to the current call data,
```

Comment:
TIOLI: I don't think this comment adds any value and just runs the risk of going out of date. The code is nice and clear as is.

Reply:
Removed.